[2022-02-12 14:00:59,822] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2022-02-11T00:00:00+00:00 [queued]>
[2022-02-12 14:00:59,873] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2022-02-11T00:00:00+00:00 [queued]>
[2022-02-12 14:00:59,874] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 14:00:59,874] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-02-12 14:00:59,877] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 14:00:59,893] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2022-02-11T00:00:00+00:00
[2022-02-12 14:00:59,899] {standard_task_runner.py:52} INFO - Started process 3761 to run task
[2022-02-12 14:00:59,908] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2022-02-11T00:00:00+00:00', '--job-id', '2', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_process.py', '--cfg-path', '/tmp/tmpv_0dr18i', '--error-file', '/tmp/tmphs9uz1xh']
[2022-02-12 14:00:59,911] {standard_task_runner.py:77} INFO - Job 2: Subtask task_1
[2022-02-12 14:01:00,040] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2022-02-11T00:00:00+00:00 [running]> on host ALE-LAPTOP.localdomain
[2022-02-12 14:01:00,719] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-02-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-11T00:00:00+00:00
[2022-02-12 14:01:00,721] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-02-12 14:01:00,722] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-02-12 14:01:00,744] {subprocess.py:75} INFO - Output:
[2022-02-12 14:01:03,749] {subprocess.py:83} INFO - Command exited with return code 0
[2022-02-12 14:01:04,022] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20220211T000000, start_date=20220212T170059, end_date=20220212T170104
[2022-02-12 14:01:04,218] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2022-02-12 14:01:04,260] {local_task_job.py:151} INFO - Task exited with return code 0
